zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -576,9 +576,9 @@ where
 
 type StepResult<DbT> = (
```

Comment:
@nuttycom would you like me to make this a struct?

Reply:
Yeah, I think that's better.

Reply:
Will do.